### PR TITLE
Check-in | Rename small-screen media

### PR DESCRIPTION
**Pull request by Mottie**
## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to Summary and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

Did you change site-wide styles, platform utilities or other infrastructure?
- [x] No
- [ ] Yes, and I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#that-sounds-normal-so-whats-the-proxy-all-about) to test the injected header scenario

## Summary

- _(Summarize the changes that have been made to the platform)_
  > The Design System Team is updating flexbox utilities in the CSS Library.
  >
  > | Current VADS Name | USWDS v3 Name |
  > | --------------------------- | ------------------------- |
  > | xsmall-screen (320px)          | mobile (320px) |
  > | small-screen (480px) | mobile-lg (480px) |
  > | small-desktop-screen (1008px) | desktop (1024px) 
  > | large-screen (1200px) | desktop-lg (1200px) |
  >
  > Unchanged names: `medium-screen`, `tablet`

- _(If bug, how to reproduce)_
- _(What is the solution, why is this the solution)_
- _(Which team do you work for, does your team own the maintenance of this component?)_
  > Benefits Decision Reviews
- _(If using a flipper, what is the end date of the flipper being required/success criteria being targeted)_

## Related issue(s)

https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/3208

## Testing done

- _Describe what the old behavior was prior to the change_
- _Describe the steps required to verify your changes are working as expected_
- _Describe the tests completed and the results_
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

check-in screen at > 480 pixels, then narrowed to less than 480 pixels shows a change in styling

## What areas of the site does it impact?

Check-in

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Comment by Mottie** [comment]
Note: I wasn't able to test these changes - I did follow the [directions in the readme](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/check-in/day-of/README.md) and ran the mock api, but I don't know how to get a valid token

**Review by powellkerry — DISMISSED** [review]

**Review by brianseek — CHANGES_REQUESTED** [review]
I tested this with a fresh yarn install and it doesn't seem like the new class is working yet.

**Comment by brianseek** [comment]
> Note: I wasn't able to test these changes - I did follow the [directions in the readme](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/check-in/day-of/README.md) and ran the mock api, but I don't know how to get a valid token

@Mottie here is a list of tokens for different scenarios https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/check-in#mock-uuids The default is `46bebc0a-b99c-464f-a5c5-560bc9eae287`

**Comment by jamigibbs** [comment]
@Mottie You might want to combine in this PR the Formation update that completes the following task from [the ticket:](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/3208)

- In formation, comment out the flexbox import in [formation/core.scss](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation/sass/core.scss#L83)

File: src/applications/check-in/sass/check-in.scss [hunk 1]
```diff
@@ -1,5 +1,6 @@
 @import "~@department-of-veterans-affairs/formation/sass/shared-variables";
 @import "~@department-of-veterans-affairs/css-library/dist/stylesheets/modules/m-action-link";
+@import "~@department-of-veterans-affairs/css-library/dist/stylesheets/utilities.css";
```
Comment (jamigibbs) [inline src/applications/check-in/sass/check-in.scss:3]: Can this update be made here in the minimal stylesheet instead? 

https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/platform/site-wide/sass/minimal.scss#L34-L35

Comment (Mottie) [reply src/applications/check-in/sass/check-in.scss]: Done

**Comment by Mottie** [comment]
@brianseek I've updated this PR and added a screenshot of what I see - please double check the changes for me. Thanks!

File: src/applications/check-in/sass/check-in.scss (hunk 1 above)
Comment (Mottie) [reply src/applications/check-in/sass/check-in.scss]: I added a GIF... it seems to work

**Comment by brianseek** [comment]
@Mottie still not seeing it. 
with `small-screen`:
Screenshot 2024-09-12 at 12 08 16 PM

with `mobile-lg`:
Screenshot 2024-09-12 at 12 08 50 PM

We have an upcoming ticket to convert these to `va-button-pair` but it will be a few sprints before we get to it.

**Comment by Mottie** [comment]
Hmm, odd. I'm seeing the yes/no buttons expanding to full width
travel-claim media query change at 480 pixels

File: src/applications/check-in/sass/check-in.scss (hunk 1 above)
Comment (Mottie) [reply src/applications/check-in/sass/check-in.scss]: So moving the import from this `check-in.scss` file into the `minimal.scss` is moving the utilities _after_ the `_flexbox.scss` imports causing the flexbox definitions to override the utilities - [see screenshots in this comment](https://github.com/department-of-veterans-affairs/vets-website/pull/31901#issuecomment-2348829056).

Moving the import back into `check-in.scss` would fix the problem for this PR, but not globally. That is until the flexbox import is removed from `formation/core.scss`. So we need to find another solution.

**Comment by jamigibbs** [comment]
I'm closing this PR in favor of this new one. Thanks for giving this a go @Mottie! 

- https://github.com/department-of-veterans-affairs/vets-website/pull/32120

**Pull request closed without merging**